nekoyume/Assets/_Scripts/UI/Widget/AdventureBoss.cs

```diff
@@ -49,6 +50,7 @@ public class AdventureBoss : Widget
         [SerializeField] private TextMeshProUGUI bossName;
         [SerializeField] private Transform bossImageParent;
         [SerializeField] private TextMeshProUGUI randomRewardText;
+        [SerializeField] private TextMeshProUGUI rewardRemainTImeText;
```

Comment:
```suggestion
        [SerializeField] private TextMeshProUGUI rewardRemainTimeText;
```

오타가 났습니다. 변수명 변경되면 SerializeField 정보도 바뀔거라 유니티쪽 수정이 불가피할 것 같네요ㅜ

Reply:
오 이거 오타는 수정하긴 해야겠네요 확인해보겠습니다~!

Reply:
c1fd3e5f416cb2c7db12976008b5d29c1527f555 여기서 수정진행완료했습니다~! 감사합니다